build-kernel.sh

```diff
@@ -9,6 +9,7 @@ set -o pipefail
 GRSECURITY="${GRSECURITY:-}"
 LINUX_VERSION="${LINUX_VERSION:-}"
 LINUX_CUSTOM_CONFIG="${LINUX_CUSTOM_CONFIG:-/config}"
+LOCALVERSION="${LOCALVERSION:-}"
```

Comment:
Looking at the diffoscope results, it looks like the maintainer, source, and homepage fields are different. It might be worth seeing if we can have more accurate values for those, eg by setting `DEBEMAIL=securedrop@freedom.press and `DEBFULLNAME="SecureDrop Team"` and passing those through to the container. 

It might be more of a hassle to change these just for the metapackages without getting into more mkdebian mods. What do you think about just overriding them for the kernel packages as well?

Reply:
(Current state for user info for kernel packages is just the build user on whatever system they get built on, which is kindof ugly, so this is something exposed by your changes rather than caused by them.)

Reply:
We were previously hardcoding it as `user@kernel-builder` in the Dockerfile, so seems like a good idea to switch. Done in the just-pushed commit and kicking off a build to verify.

Reply:
```
user@sd-kernel-builder:~/git-kernel-builder$ apt show ./build/securedrop-grsec_5.15.68-grsec-securedrop-1_amd64.deb
Package: securedrop-grsec
Version: 5.15.68-grsec-securedrop-1
Priority: optional
Section: admin
Source: linux-upstream
Maintainer: SecureDrop Team <securedrop@freedom.press>
Installed-Size: 15.4 kB
Depends: linux-image-5.15.68-grsec-securedrop, intel-microcode, paxctld
Homepage: https://securedrop.org/
Download-Size: 3,012 B
APT-Sources: /home/user/git-kernel-builder/build/securedrop-grsec_5.15.68-grsec-securedrop-1_amd64.deb
Description: Metapackage providing a grsecurity-patched Linux kernel for use
 with SecureDrop. Depends on the most recently built patched kernel maintained
 by FPF. Package also includes sysctl and PaX flags calls for GRUB.

user@sd-kernel-builder:~/git-kernel-builder$ apt show ./build/linux-image-5.15.68-grsec-securedrop_5.15.68-grsec-securedrop-1_amd64.deb
Package: linux-image-5.15.68-grsec-securedrop
Version: 5.15.68-grsec-securedrop-1
Priority: optional
Section: kernel
Source: linux-upstream
Maintainer: SecureDrop Team <securedrop@freedom.press>
Installed-Size: 343 MB
Homepage: https://securedrop.org/
Download-Size: 59.2 MB
APT-Sources: /home/user/git-kernel-builder/build/linux-image-5.15.68-grsec-securedrop_5.15.68-grsec-securedrop-1_amd64.deb
Description: Linux kernel, version 5.15.68-grsec-securedrop
 This package contains the Linux kernel, modules and corresponding other
 files, version: 5.15.68-grsec-securedrop.
```